objects/Tank.java

```diff
@@ -171,4 +141,25 @@ public void move(Commands.Direction cmd)
     }
     return bitmap;
   }
-}
+
+  public boolean isCollision(Renderer mainRenderer, int rowPos, int colPos) {
```

Comment:
I think this is a helper function, which should be private.

Reply:
Thanks for the suggestion I missed that. I have made it private now